.github/workflows/uninstall_install_sandbox.yaml

```diff
@@ -58,9 +71,22 @@ jobs:
           namespace: ${{ secrets.OPENSHIFT_NAMESPACE }}
 
       - name: Install Traction via Helm
-        continue-on-error: true
+        continue-on-error: false
         run: |
           helm repo add traction https://bcgov.github.io/traction 
           helm upgrade --install traction-sandbox traction/traction \
           -f ./helm-values/traction/values-sandbox.yaml \
-          -n ${{ secrets.OPENSHIFT_NAMESPACE }} --wait --timeout=7m
\ No newline at end of file
+          -n ${{ secrets.OPENSHIFT_NAMESPACE }} --wait --timeout=7m
+
+      - name: Rocket.Chat Notification
+        uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@master
+        if: always()
+        with:
+          type:  ${{ job.status }}
+          job_name: '*Install Traction Sandbox*'
+          mention: 'here'
+          mention_if: 'failure'
+          channel: '#ditp-gha-notifications'
+          url: ${{ secrets.ROCKETCHAT_WEBHOOK }}
+          commit: true
+          token: ${{ secrets.ROCKETCHAT_TOKEN }}
```

Comment:
`token: ${{ secrets.ROCKETCHAT_TOKEN }}` is not required if the `secrets.ROCKETCHAT_WEBHOOK` contains the token, which in our case it does.  Refer to your previous work with this action here; https://github.com/bcgov/von-bc-registries-audit/blob/main/.github/workflows/actions/deploy/action.yml#L83-L93